### DISCUSSION TITLE
Obok DeDRM v10.0.3 error: Could not find Kobo LIbrary

### DISCUSSION THREAD

**Discussion by piet2209** [apprenticeharper/DeDRM_tools#2183] [Q&A]
I have used Obok DeDRM v10.0.3 as a plugin in Calibre 6.8.0 the last months without any problems.
Yesterday I upgraded Calibre to 6.10.0. This new version of Calibre works fine but from the Obok plugin I receive the error: could not find Kobo library. I deleted Calibre 6.10.0 and re-installed 6.8.0 again but unfortunately I still receive the same error message from Obok.
Any thoughts what is wrong?
Thanks in advance and merry Christmas!

**Comment by ElleKayEm** [comment]
Might want to post this over in [noDRM's repo](https://github.com/noDRM/DeDRM_tools)  instead of here in Apprentice Harper's.

Unfortunately I have no answer for the question.  For whatever it's worth, Obok 10.0.3 is still working for me with calibre 6.10 on Windows 10 and Kobo Desktop.
